### PR TITLE
Stimuli in +stimulus 

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**
Working stimuli:
* stimulus.Varma
* stimulus.Matisse
* stimulus.Matisse2
* stimulus.Clip

### CODE REVIEW

File: matlab/schemas/+stimulus/Clip.m [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+-----
+-> stimulus.MovieClip
+cut_after              : float           # (s) cuts off after this duration
+%}
+
+classdef Clip < dj.Manual & stimulus.core.Visual
+    
+    properties(Constant)
+        variation = '1'
+        movie_dir = '/Users/dimitri/stimuli'
```
Comment (ecobost) [inline matlab/schemas/+stimulus/Clip.m]: Should there be a different default here?

File: matlab/schemas/+stimulus/Matisse2.m [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                self.flip(opts)
+                opts.checkDroppedFrames = true;
+                Screen('close',tex)
+            end
+        end
+    end
+end
+
+
+function img = upscale(img, factor, ori, coherence, nframes, speed)
```
Comment (ecobost) [inline matlab/schemas/+stimulus/Matisse2.m]: These functions could be indexed. It looks nicer

Comment (dimitri-yatsenko) [inline matlab/schemas/+stimulus/Matisse2.m]: Did you mean "indented"?  I use MATLAB's indentation rules, which are enforced when you mark all text (Ctrl-A) and then press Ctrl-I.   These are function that are external to the class and they are not indented in MATLAB.

File: matlab/schemas/+stimulus/Clip.m (hunk 1 above)
Comment (dimitri-yatsenko) [inline matlab/schemas/+stimulus/Clip.m]: Yes, it should be `~/stimuli` but PsychToolbox does not understand `~`.  we should ask Manolis what it this should be.

Comment (ecobost) [inline matlab/schemas/+stimulus/Clip.m]: In +stimulus/MovieClip.m you put movie_dir='movies', maybe movie_dir='stimuli' is good here

**Pull request merged**